web/app/themes/clarity/inc/admin/prior-party/prior-party-banner-admin.php

```diff
@@ -259,7 +272,7 @@ public function page(): void
 
                     // links
                     $link_admin = get_edit_post_link($post->ID);
-                    $link_view = get_permalink($post->ID) . '?time_context=' . $stop->format('U');
+                    $link_view = get_permalink($post->ID) . '?' . http_build_query($link_view_queries);
```

Comment:
What would the expected outcome of this be?

My understanding either is:
```php
?show_inactive=''
```
or,
```php
?time_context=123456789
```

Is this right?

Reply:
It's 0 if we're just 'Viewing' an active banner... seeing the banner the same as a logged out user.
Just `show_inactive` for looking at the Truss banner for example, no time travel necessary ;)
Both for the Sunak banner if we were 'Previewing' it last week.